modules/navier_stokes/doc/content/source/hdgkernels/NavierStokesHDGKernel.md

```diff
@@ -0,0 +1,21 @@
+# NavierStokesHDGKernel
+
+This class implements the steady incompressible Navier-Stokes equations. These include the conservation of mass:
+
+!equation id=mass-eq
+\nabla \cdot \vec{u} = 0
+
+where $\vec{u}$ is the velocity and conservation of momentum:
+
+!equation id=momentum-eq
+\nabla \cdot \left(\rho \vec{u} \otimes \vec{u}\right) - \nabla \cdot \left(\mu
+\nabla\vec{u}\right) + \nabla p = 0
+
+where $\rho$ is the density, $\mu$ is the dynamic viscosity and $p$ is the pressure. This class uses
+the hybridization laid out in [!citep](nguyen2011implicit).
+
```

Comment:
would be good to mention the integration by parts since it matters for implementing more BCs

Reply:
also the weak forms for the equations
this page can then stand-in for the hdgns.md we will eventually want to link to from the main table of all NS discretizations we have

Reply:
umm I guess the reference to nguyen is good enough for the equations

Reply:
umm I guess the reference to nguyen is good enough for the equations

Reply:
it's not open access but I can find it here
https://acdl.mit.edu/AIAA2010_362.pdf